README.md

```diff
@@ -46,7 +44,9 @@ jobs:
     steps:
       - checkout
       - datadog-static-analysis/analyze:
-          service: my-service
+          service: "my-service"
+          env: "ci"
+          site: datadoghq.com
```

Comment:
Does this need to change with site region?

Reply:
```suggestion
          site: {{< region-param key="dd_site" code="true" >}}
```